source/features/linkify-user-location.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {wrap} from '../libs/dom-utils';
+import features from '../libs/features';
+
+function addLocation(baseElement?: HTMLElement): void {
+	const locationIcon = select('.octicon-location', baseElement)!;
+
+	if (!locationIcon) {
+		return;
+	}
+
+	const location = locationIcon.parentElement!.textContent!.trim();
+	const googleMapsLink = `https://www.google.com/maps/place/${encodeURIComponent(location)}/`;
```

Comment:
Because the users location can be anything, the correct should be `https://www.google.com/maps/search/?api=1&query=`. See documentation: https://developers.google.com/maps/documentation/urls/guide#search-action